shared/whats-new/index.tsx

```diff
@@ -0,0 +1,156 @@
+import React from 'react'
+import * as Kb from '../common-adapters'
+import * as Styles from '../styles'
+import * as Types from '../constants/types/whats-new'
+import {VersionProps} from './versions'
+
+type Props = {
+  onBack: () => void
+  onNavigate: (props: {}, selected: string) => void
+  onNavigateExternal: (url: string) => void
+  seenVersions: {[key: string]: boolean}
+  currentVersion: Types.CurrentVersion
+  lastVersion: Types.LastVersion
+  lastLastVersion: Types.LastLastVersion
+  noVersion: string
+  Current?: React.ComponentType<VersionProps>
+  Last?: React.ComponentType<VersionProps>
+  LastLast?: React.ComponentType<VersionProps>
+}
+
+// Need to switch the order of the scroll view on mobile and desktop so that contentBackground will fill the entire view
+const Wrapper = ({children}: {children: React.ReactNode}) =>
+  Styles.isMobile ? (
+    <Kb.Box2 direction="vertical" alignItems="flex-start" alignSelf="flex-start" fullHeight={true}>
```

Comment:
seems like the first two containers can be shared across desktop/mobile, just use `isMobile` tests on the `style` and `fullWidth`?